Ghpr.CouchDb/Mappers/ItemInfoDtoMapper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Ghpr.Core.Common;
+using Ghpr.CouchDb.Entities;
+
+namespace Ghpr.CouchDb.Mappers
+{
+    public static class ItemInfoDtoMapper
+    {
+        public static ItemInfo MapTestRunInfo(this ItemInfoDto itemInfoDto)
```

Comment:
This can be done with one line of code of  Automapper. If you do not want external dependency then you can crate your own method reflection-based